storage/aptosdb/src/pruner/mod.rs

```diff
@@ -89,17 +116,34 @@ impl Pruner {
     /// (For tests only.) Notifies the worker thread and waits for it to finish its job by polling
     /// an internal counter.
     #[cfg(test)]
-    pub fn wake_and_wait(&self, latest_version: Version) -> Result<()> {
+    pub fn wake_and_wait(
+        &self,
+        latest_version: Version,
+        pruner_index: usize,
+    ) -> anyhow::Result<()> {
+        use std::{
+            thread::sleep,
+            time::{Duration, Instant},
+        };
+
         self.wake(latest_version);
 
-        if latest_version > self.historical_versions_to_keep {
-            let least_readable_version = latest_version - self.historical_versions_to_keep;
+        if latest_version > self.state_store_prune_window
+            || latest_version > self.default_prune_window
+        {
+            let least_readable_state_store_version = latest_version - self.state_store_prune_window;
             // Assuming no big pruning chunks will be issued by a test.
-            const TIMEOUT: Duration = Duration::from_secs(10);
+            const TIMEOUT: Duration = Duration::from_secs(60);
```

Comment:
Is this necessary? We would try to avoid slow unit tests..

Reply:
Thanks for noticing, this is not needed. I added this for some debugging purpose.